.github/workflows/reuse.yml

```diff
@@ -0,0 +1,17 @@
+# synced from @nextcloud/android-config
```

Comment:
Shouldn't it be synced from https://github.com/nextcloud/.github?

Reply:
Do you mean that reuse.yml in `nextcloud/android-config` should be moved to `nextcloud/.github` ?

Reply:
Good point, makes sense I guess
This one slipped through given that formerly I only did these changes for Android Repos